java/java.lsp.server/nbproject/project.xml

```diff
@@ -126,7 +132,15 @@
                     <compile-dependency/>
                     <run-dependency>
                         <release-version>1</release-version>
-                        <specification-version>1.82</specification-version>
+                        <specification-version>1.81</specification-version>
```

Comment:
Old version, was `1.82`. In fact this PR adds an API to db.explorer, so should even increase + apichanges & etc.